samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -58,6 +64,8 @@ var AppController = function(loadingParams) {
   this.remoteVideo_ = $(UI_CONSTANTS.remoteVideo);
   this.videosDiv_ = $(UI_CONSTANTS.videosDiv);
   this.roomLinkHref_ = $(UI_CONSTANTS.roomLinkHref);
+  
+  this.inCall_ = false;
```

Comment:
I think a better way is to unwire the keypressed callback when it's out of the call view